SwSpotify/spotify.py

```diff
@@ -112,13 +112,30 @@ def get_info_mac():
     return a[3], a[1]
 
 
-def current():
-    if sys.platform.startswith("win"):
-        return get_info_windows()
-    elif sys.platform.startswith("darwin"):
-        return get_info_mac()
+def get_info_web():
+    from SwSpotify.web_data import WebData
+    import SwSpotify.web_server
+    import os
+    subprocess.Popen([sys.executable, os.path.join(os.path.dirname(__file__) + "/server_shutdown.py")])
+    SwSpotify.web_server.run()
+    if WebData.track == "":
+        raise SpotifyClosed
+    elif WebData.playState == "Play":
+        raise SpotifyPaused
     else:
-        return get_info_linux()
+        return WebData.track, WebData.artist
+
+
+def current():
+    try:
+        if sys.platform.startswith("win"):
+            return get_info_windows()
+        elif sys.platform.startswith("darwin"):
+            return get_info_mac()
+        else:
+            return get_info_linux()
+    except SpotifyClosed:
```

Comment:
You probably want to catch SpotifyNotRunning, if the Spotify desktop app is open but the music is playing from the web player, SpotifyClosed would not be raised

Reply:
Not really, spotify web player and client is synchornizing, so when you play on a web player, desktop app is also displaying the same info. It might be a problem while using 2 different accounts on web and client, but that's quite an edge case. But If I changed to SpotifyNotRunning, then if client would return spotify paused, the get_info_web would return spotify closed instead spotify paused

Reply:
Are you on Windows? That's what happens on Windows generally. In that case, what if the native app is paused so then it checks the extension and if that is closed too then that's that?

Reply:
You mean, if native returns paused and extension returns closed, then raise SpotifyPaused?

Reply:
No like if native returns paused and extension returns closed then return paused. But if native is paused but extension returns data then use that

Reply:
Ok, got it